docs/Color-Picker/Color-Picker.md

```diff
@@ -1 +1,24 @@
-TODO
\ No newline at end of file
+## Requirements
+
+### Part 1: App-Only
+* The audience will listen to the orchestra play four pieces of music.
+* Each audience member will use the app to pick from six available colors (RGB, CMY).
+* It's possible that we will want audience members of the final rehearsal to be able to choose emotions instead.
+* The audience member can pick a new color at any time, but the app will give a gentle reminder every 20 seconds.
+* During the intermission (after the four songs), each audience member's app will display both their own choices as well as the overall audience distribution. Let's call this the "results" screen.
+* The app will move from one song to the next as the event progresses without user input
+
+### Part 2: App + Central Visualization
+* In the third section, the audience will listen to the orchestra play the Coriolan overture.
+* The audience will be randomly split into two groups.
+    * In group A, the functionality is the same as the app-only section.
+    * In group B, the audience member will instead be picking emotions: Joy, sadness, anger, fear, [disgust].
+* There will also be a central display in the event venue visualizing the audience's choices as they come in. This should function like a "mood ring" in the sense that it shows the most recent audience choices.
```

Comment:
We should build our own, but there's a possibility that some super-awesome team might contribute one for the event. If that happens, ours can still be used in the app.

Reply:
Is the central display an existing piece of equipment at the concert hall, or is it something that we're planning to set up ourselves? If the former, do we know whether there's some API we'll need to use to communicate with it? Or will it be as simple as plugging an input cable into some device to mirror app display?

Reply:
That's a good question. I'm hoping it's the latter. Arseny mentioned "screens".